src/components/CustomField.tsx

```diff
@@ -20,6 +20,7 @@ import { Entity } from '$app/common/hooks/useEntityCustomFields';
 interface Props extends InputCustomFieldProps {
   fieldOnly?: boolean;
   noExternalPadding?: boolean;
+  selectMenuPosition?: 'fixed' | 'absolute';
```

Comment:
What are the differences here? From what I can see, we always use “fixed”. Is absolute old default?

Reply:
@beganovich Actually, we don't set anything by default - the `menuPosition` component prop is left to use the package's default setting. It accepts 'fixed' and 'absolute' values. While we NEVER use 'absolute', I initially included it since the package accepts that value. However, to avoid confusion, I've removed 'absolute' and kept only 'fixed'.